_posts/2020-03-05-git-secret.md

```diff
@@ -119,7 +120,7 @@ echo $GPG_PRIVATE_KEY | tr ',' '\n' > ./private_key.gpg
 
 ## Environment Variables and Configuration
 
-You can configure the version of gpg used, or the extension your encrypted files use, to suit your workflow better.
+You can configure the version of `gpg` used, or the extension your encrypted files use, to suit your workflow better.
 To do so, just set the required variable to the value you need.
 This can be done in your shell environment file or with each `git-secret` command.
```

Comment:
Please say something like:

See the man page for `git-secret` for an explanation of the environment variables `git-secret` uses.

Reply:
I should have read more carefully before making my last suggestion; I see that git-secret's env vars are described just below here. Can you please change the sentence to say:

"See below, or the man page for git-secret, for an explanation of the environment variables git-secret uses."

Then I'll go ahead and merge this PR.